scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.scio
 
+import com.google.bigtable.repackaged.com.google.cloud.grpc.BigtableClusterUtilities
```

Comment:
This is shaded. Is it intended for public use? At least I'd add a wrapper or type alias in https://github.com/spotify/scio/blob/master/scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala#L40